pkg/cmd/init/exec.go

```diff
@@ -49,6 +51,12 @@ func (o *Options) validate() error {
 	if installed {
 		return fmt.Errorf("hub already initialized")
 	}
+	if len(o.registry) == 0 {
```

Comment:
As there is a default, this is not needed to check o.registry nor o.version

Reply:
but i guess it's fine to validate them in case of unexpected misconfiguration?